build.yaml

```diff
@@ -251,42 +251,15 @@ common_cli:
     # Set up the new venv.
     virtualenv -p python2.7 --clear ${venv}
     . ${venv}/bin/activate
-    # Report the version of Python we're using, to aid debugging.
+    # Report the version of Python and pip we're using, to aid debugging.
     ${venv}/bin/python --version
-
-  setup_pip_cache: &setup_pip_cache |
-    # exports the PIP_INDEX_URL, TRUSTED_HOST variables to the environment.
-    # The /tmp/pip.sh file is copied to the Jenkins Slave by the Jenkins Master
-    # during the bootstrapping of the slave.
-    # This file is located in /etc/jenkins_slave on the Master box.
-    # These variables contain the PIP URL and IP address of our caching server.
-    if [ -e /tmp/pip.sh ]; then
-      . /tmp/pip.sh
-    fi
```

Comment:
I've removed the devpi stuff here too because I couldn't install the latest version of pip from devpi.
And pip 7.1.0 has a weird restriction which prevents you installing two requirements that contain the same package.